setup.cfg

```diff
@@ -35,6 +36,8 @@ project_urls =
 
 [options]
 packages = find:
+install_requires =
+    aiofiles>=22.1.0
```

Comment:
Could you make this an optional dependency? (using an extra, so people can depend on `whitenoise[asgi]`)